omero/sysadmins/windows-migration.txt

```diff
@@ -0,0 +1,62 @@
+Migration from OMERO 5.2 on Windows to OMERO 5.3 on Linux
+=========================================================
+
+Backup (Windows)
+----------------
+
+- Create a dump of the OMERO SQL database, either using
+  `pgAdmin <https://www.pgadmin.org/>`_ or the :command:`pg_dump` command line
+  program::
+
+    pg_dump -Fc -f omero52.pg_dump omero_database
+
+- Please keep a pristine copy of your Windows OMERO server, until you are
```

Comment:
"binary repository" is the term used under backup and restore:

```
omero/sysadmins/server-backup-and-restore.txt:store <unix/server-binary-repository>`. The entire contents of ``/OMERO`` should be
```

Perhaps the same link could be used?